hw1/bashcommands.py

```diff
@@ -0,0 +1,79 @@
+from collections import deque
+
+
+def for_each_argument(f):
+    def f_for_many(*many_args):
+        for arg in many_args:
+            f(arg)
+
+    return f_for_many
+
+
+def wc(*filepaths):
+    total_lines = 0
+    total_words = 0
+    total_bytes = 0
+    for filepath in filepaths:
+        cur_lines = 0
+        cur_words = 0
+        cur_bytes = 0
+        try:
+            with open(filepath) as f:
+                for line in f:
+                    cur_lines += 1
+                    cur_words += len(line.split())
+                    cur_bytes += len(bytes(line, encoding="utf8"))
+            total_lines += cur_lines
+            total_words += cur_words
+            total_bytes += cur_bytes
+            print(f"lines:{cur_lines} words:{cur_words} bytes:{cur_bytes} for {filepath}")
```

Comment:
Please, add spaces after ":" and commas between items, just so that it looks nicer.